e2e/proxy_test.go

```diff
@@ -102,3 +102,204 @@ func TestProxiedEnvironment(t *testing.T) {
 
 	t.Logf("%s: test complete", time.Now().Format(time.RFC3339))
 }
+
+// TestCustomCIDR tests the installation with an alternate CIDR range
+func TestCustomCIDR(t *testing.T) {
+	t.Parallel()
+	tc := cluster.NewTestCluster(&cluster.Input{
+		T:                   t,
+		Nodes:               4,
+		Image:               "debian/12",
+		LicensePath:         "license.yaml",
+		EmbeddedClusterPath: "../output/bin/embedded-cluster",
+	})
+	defer cleanupCluster(t, tc)
+	t.Log("Proxied infrastructure created")
+
+	// bootstrap the first node and makes sure it is healthy. also executes the kots
+	// ssl certificate configuration (kurl-proxy).
+	t.Logf("%s: installing embedded-cluster on node 0", time.Now().Format(time.RFC3339))
+	line := []string{"single-node-proxy-install.sh"}
```

Comment:
is this supposed to be using a proxy install?

Reply:
no, but the proxy install script already supported arbitrary additional args
I'll fix it

Reply:
it is not using a proxy install despite the name of the script here